verl/utils/reward_score/gsm8k.py

```diff
@@ -20,12 +20,12 @@ def extract_solution(solution_str, method="strict"):
 
     if method == "strict":
         # this also tests the formatting of the model
```

Comment:
does this help improve the answer exaction success rate? if so we need to note that in the release note so that users are aware